infra/tf/main.tf

```diff
@@ -71,42 +70,41 @@ resource "digitalocean_droplet" "bootstrap-node" {
   size       = var.droplet_size
   backups    = false
   monitoring = true
-  user_data  = file(format("%s/bootstrap_node.yml", path.module))
+  user_data  = file("${path.module}/configs/bootstrap_node.yml")
```

Comment:
Any benefit to this change?

Reply:
cleaner code, and we're not doing any special formatting to really warrant using the full format when just using variable replacement works fine

Reply:
I don’t agree that it’s cleaner code

Reply:
It’s more cognitive load to figure out why the change was made vs subjective cleaner code change

Reply:
Let’s try to go over the PR and revert such subjective cleaner code changes

Reply:
Okay sure. Although the reason was to also follow the same coding style with line 158 in the same file; which would be hard to read with the additional function call vs inline variable. I can still revert and modify the other line if you think it's better